contents/blog/posthog-vs-logrocket.md

```diff
@@ -0,0 +1,767 @@
+        <tr>
+            <td><strong>Discord</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Receive notifications about new data</td>
+        </tr>
+        <tr>
+            <td><strong>MS Teams</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Receive notifications about new data</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+## Compliance
+
+Regulatory compliance can be make-or-break for many teams, especially those operating in fields such as healthcare or the financial sector. Luckily, LogRocket and PostHog both offer a variety of ways to comply with such regulations.
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+            <td className="w-3/12"></td>
+            <td><strong>LogRocket</strong></td>
+            <td><strong>PostHog</strong></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>GDPR ready</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td><strong>Data anonymization</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td><strong>Cookie-less tracking option</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td><strong>SOC 2 certified</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        </tr>
+        <tr>
+            <td><strong>SAML/SSO available</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td><strong>2FA available</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+> PostHog is not currently SOC 2 certified, but is currently being audited in order to complete this certification.
```

Comment:
```suggestion
> PostHog is not currently SOC 2 certified, but is [currently being audited](/handbook/company/security) in order to complete this certification.
```
I assume this will update and keep people up to date if they care

Reply:
Yeah, when we clear SOC 2 we should do a sweep and correct all sources. @andyvan-ph